sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Common/MetricsTest.cs

```diff
@@ -414,6 +414,8 @@ public void Complexity()
                 .Should().Be(2);
             Complexity(AnalyzerLanguage.CSharp, "class MyClass { void MyMethod() { var t = null ?? 0; } }")
                 .Should().Be(2);
+            Complexity(AnalyzerLanguage.CSharp, "class MyClass { void MyMethod(int? t) { t ??= 0; } }")
```

Comment:
do you want to add a test with cognitive complexity on this PR as well?

Reply:
There is one already: https://github.com/SonarSource/sonar-dotnet/blob/b6a7003b67dbd0ae7dae927cb8950e533f923d06/sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/CognitiveComplexity.cs#L173-L178